apisix/plugins/basic-auth.lua

```diff
@@ -39,6 +44,10 @@ local consumer_schema = {
     properties = {
         username = { type = "string" },
         password = { type = "string" },
+        hide_auth_header = {
+            type = "boolean",
+            default = true,
```

Comment:
```suggestion
            default = false,
```
Need to discuss it in maillist if use `true` as default value